client/src/components/Countdown/Countdown.js

```diff
@@ -0,0 +1,46 @@
+import "./Countdown.modules.scss";
+import React from "react";
+import { useSelector } from "react-redux";
+
+import Typography from "@material-ui/core/Typography";
+import LinearProgress from "@material-ui/core/LinearProgress";
+
+export default function Countdown(props) {
+    const tasks = useSelector((state) => state.tasks);
+
+    let countdownVal = -1, progressVal = -5, countdownMins = 0, countdownSecs = 0;
+    const runningIndex = tasks.findIndex((elem) => elem.running);
+    if (runningIndex !== -1) {
+        countdownVal = tasks[runningIndex].pomodoro_duration - tasks[runningIndex].secondsElapsed;
```

Comment:
can we encapsulate this into a util function which take in pomodoro_duration and secondsElapsed and return progressVal?